generators/database-changelog-liquibase/incremental-liquibase.spec.mts

```diff
@@ -641,4 +676,60 @@ relationship OneToOne {
       expect(runResult.getSnapshot('**/src/main/resources/config/liquibase/**')).toMatchSnapshot();
     });
   });
+  context('entities with/without byte fields should create fake data', () => {
+    [
+      {
+        entity: jdlApplicationEntitieWithByteTypes,
+        bytesFields: true,
+        testContent:
+          '1;Integratio;13255;../fake-data/blob/hipster.txt;../fake-data/blob/hipster.png;image/png;../fake-data/blob/hipster.png;image/png;Laptop;89202',
```

Comment:
I prefer snapshot instead of fixed content.
If we move some files, the faker seed can change and the test must be updated manually.